### PR TITLE
[Bug Fix] Fix miscellaneous memory leaks related to EQApplicationPacket and it's pBuffer

### PR DESCRIPTION
**Pull request by Quintinon**
Have not had a chance to even compile this let alone run it, hence the draft PR status.

### CODE REVIEW

**Comment by Quintinon** [comment]
GitHub compiled it for me, so that is passing at least. Still have not had a chance to run this on my server, however I've finished my initial pass through the code base (I looked at every instance of the word new).

I still have some notes on rabbit trails for follow up on but any findings from those will be in a separate PR.

Feedback is welcome at this point on anything. I tried to be as minimally invasive as possible on the changes since there were a lot in a single PR.

File: zone/gm_commands/reload.cpp [hunk 1]
```diff
@@ -61,7 +61,7 @@ void command_reload(Client *c, const Seperator *sep)
 		return;
 	}
 
-	auto pack = new ServerPacket;
+	ServerPacket* pack;
```
Comment (hgtw) [inline zone/gm_commands/reload.cpp:64]: Initialize this with `= nullptr` since it might contain a non-null garbage value

Comment (Quintinon) [reply zone/gm_commands/reload.cpp]: Done

**Review by hgtw — APPROVED** [review]

File: zone/trading.cpp [hunk 2]
```diff
@@ -1151,8 +1152,8 @@ void Client::Trader_EndTrader() {
 			}
 
 			safe_delete(outapp);
-			safe_delete(gis);
 		}
+		safe_delete(gis);
```
Comment (hgtw) [inline zone/trading.cpp:1156]: Just undo this change and push a new commit to this pr's branch to resolve the #2266 duplicate

Comment (Quintinon) [reply zone/trading.cpp]: Done

**Comment by Akkadius** [comment]
Conflicts

**Comment by Akkadius** [comment]
Handled conflicts, awaiting build
